src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { BrowserRouter } from 'react-router-dom'
+import { mount } from 'enzyme'
+import { TableDetail } from '../TableDetail'
+import i18n from '../../../../../../i18nTestConfig'
+import summarize from '../../../../../../rippled/lib/txSummary'
+import mockEscrowFinish from './mock_data/EscrowFinish.json'
+
+const createWrapper = (tx: any) =>
```

Comment:
Are any of these createWrapper functions equivalent across the various tests? (maybe it makes sense to have that be in a helper function?

Reply:
Some are similar.  Its more just a naming convention.